SharpSource/SharpSource.Test/Helpers/CSCodeFix+Test.cs

```diff
@@ -23,11 +31,31 @@ public Test()
                     return solution;
                 });
 
+                // After netframework the runtime is split in System.Runtime and System.Private.CoreLib
+                // All references appear to return System.Private.CoreLib so we'll have to manually insert the System.Runtime one
+                TestState.AdditionalReferences.Add(SystemPrivateCoreLibPath);
+                TestState.AdditionalReferences.Add(GetDllDirectory("System.Runtime.dll"));
+                TestState.AdditionalReferences.Add(typeof(Microsoft.AspNetCore.Mvc.FromBodyAttribute).Assembly.Location);
+                TestState.AdditionalReferences.Add(typeof(System.Net.Http.HttpClient).Assembly.Location);
+                TestState.AdditionalReferences.Add(typeof(IHttpClientFactory).Assembly.Location);
+                TestState.AdditionalReferences.Add(typeof(Console).Assembly.Location);
+                TestState.AdditionalReferences.Add(typeof(Enumerable).Assembly.Location);
+                
+                // Initialized with an empty object so the underlying test framework doesn't auto-inject all the netcoreapp3.1 references
+                TestState.ReferenceAssemblies = new ReferenceAssemblies("net7.0");
```

Comment:
`ReferenceAssemblies.Net.Net70` is better and is the preferred approach. I don't quite understand what the problem with it is.

Reply:
Ah, that would make sense! The default option added the netcoreapp3.1 references which I only figured out through digging in the internals and didn't spot an alternative for it. It looks like I only have up to `Net.Net60` available so I guess I'll need to bump the version. I'll switch to that approach instead 👍

Reply:
It seems Net70 is in 1.1.2 (beta and not published on NuGet.org - I opened https://github.com/dotnet/roslyn-sdk/issues/1047)

For now, Net60 should probably do the work.